website/content/docs/security/acl/acl-rules.mdx

```diff
@@ -10,54 +10,104 @@ description: >-
 
 # ACL Rules
 
--> **1.4.0 and later:** This document only applies in Consul versions 1.4.0 and later. The documentation for the legacy ACL system is [here](/docs/acl/acl-legacy).
+This topic describes how to configure rules for Consul's access control list (ACL) system. The ACL system enables you to control access to data and APIs. Refer to the [ACL system documentation](/docs/acl/acl-system) to learn more about ACLs. 
 
-Consul provides an optional Access Control List (ACL) system which can be used
-to control access to data and APIs. To learn more about Consul's ACL review the
-[ACL system documentation](/docs/acl/acl-system)
-
-A core part of the ACL system is the rule language, which is used to describe the policy
-that must be enforced. There are two types of rules: prefix based rules and exact matching
-rules.
+-> **1.4.0 and later:** This topic applies to Consul versions 1.4.0 and later. Refer to the [legacy ACL system documentation](/docs/acl/acl-legacy) for older versions of Consul.
 
 ## Rule Specification
 
-Rules are composed of a resource, a segment (for some resource areas) and a policy
-disposition. The general structure of a rule is:
+ACL rules describe the level of access to resources. A rule is composed of a resource declaration and an access level defined with the `policy` keyword and a [policy disposition](#policy-dispositions). The following syntax describes the basic structure of a rule:
 
-```text
-<resource> "<segment>" {
+```hcl
+<resource> {
```

Comment:
None of the rules actually look like this right? I think we might be better without this example since we have two examples of the block and non-block rules right below this.